### PR TITLE
Badges and trailing spaces

### PR DESCRIPTION
**Pull request by otreblan**
[AUR version](https://aur.archlinux.org/packages/i3-agenda/)
[PyPI](https://pypi.org/project/i3-agenda/)

Added badges to the README and deleted the trailing spaces.

### CODE REVIEW

**Comment by rosenpin** [comment]
Thanks :)  
The trailing 2 spaces at end of lines are for markdown line breaks actually :p

**Comment by otreblan** [comment]
The 2 spaces were replaced by `\`

**Comment by rosenpin** [comment]
Oh nice didn't know it worked like this

File: README.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
+[![AUR version](https://img.shields.io/aur/version/i3-agenda?style=flat-square&logo=arch-linux)](https://aur.archlinux.org/packages/i3-agenda/)
+[![PyPI](https://img.shields.io/pypi/v/i3-agenda?style=flat-square&logo=python)](https://pypi.org/project/i3-agenda/)
 # What is this?
-It's a script that communicates with Google's calendar API, it will go through all your calendars and print the next calendar event details
```
[the remark this answers is not in the document]
Comment (rosenpin) [reply README.md:2]: Actually seems like this is the only place

[the remark this answers is not in the document]
Comment (otreblan) [reply README.md:2]: Done

**Pull request merged**